osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -176,8 +179,10 @@ protected SongSelect()
         }
 
         [BackgroundDependencyLoader(permitNulls: true)]
-        private void load(BeatmapManager beatmaps, AudioManager audio, DialogOverlay dialog, OsuGame osu, OsuColour colours)
+        private void load(BeatmapManager beatmaps, AudioManager audio, DialogOverlay dialog, OsuGame osu, OsuColour colours, GameBeatmap beatmap)
         {
+            this.beatmap = beatmap.GetBoundCopy();
```

Comment:
shouldn't we be constructing a bindable in this class instead? this is making song select reliant on a beatmap being available (probably not the case in a song select testcase).

Reply:
also is there a reason not for using `OsuScreen`'s?